cdc/api/v2/processor.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v2
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/pingcap/tiflow/cdc/model"
+)
+
+// listProcessors lists all processors in the TiCDC cluster
+// @Summary List processors
+// @Description list all processors in the TiCDC cluster
+// @Tags processor
+// @Accept json
+// @Produce json
+// @Success 200 {array} model.ProcessorCommonInfo
+// @Failure 500,400 {object} model.HTTPError
+// @Router	/api/v1/processors [get]
+func (h *OpenAPIV2) listProcessors(c *gin.Context) {
+	ctx := c.Request.Context()
+	infos, err := h.capture.StatusProvider().GetProcessors(ctx)
+	if err != nil {
+		_ = c.Error(err)
+		return
+	}
+	resps := make([]*model.ProcessorCommonInfo, len(infos))
```

Comment:
follow the API style, use `ListResponse`

Reply:
changed to `ListResponse`